code/modules/reagents/reagent_dispenser.dm

```diff
@@ -513,6 +532,19 @@
 	desc = "Originally used to store liquids & powder. It is now used as a source of comfort. This one is made of wood."
 	health = 30
 
+/obj/structure/reagent_dispensers/cauldron/barrel/wood/attackby(obj/item/weapon/W, mob/user)
+	if (iscrowbar(W))
+		var/obj/item/tool/crowbar/C = W
+		to_chat(user, "<span class='notice'>You begin deconstructing the [src].</span>")
```

Comment:
it's more proper to use \the [src] here (and all other instances)

this will avoid "You begin deconstructing the the cauldron" but it will allow "You begin deconstructing the Witch's Cauldron" (if given a proper name).